src/desktop/apps/feature/routes.coffee

```diff
@@ -1,14 +1,16 @@
 Feature = require '../../models/feature.coffee'
 
-@index = (req, res) ->
+@index = (req, res, next) ->
   new Feature(id: req.params.id).fetch
     error: res.backboneError
     success: (feature) ->
-      res.locals.sd.FEATURE = feature
-      res.locals.sd.TAB = req.params.tab
-      res.render 'templates/index',
-        feature: feature
+      if req.params.id == "alserkal-art-week"
```

Comment:
Would update this to check for a field on the `feature`. This is just for testing purposes.

Reply:
Ahh @damassi I feel like this flow won't work with an all-reaction routing setup, but I think we need it in order to preserve the URL handling.

I think it'll be worth us trying to deprecate the old feature pages (hopefully the work that we do will make that easy) as soon as possible. 

But should I remove `featureRoutes` from `getAppRoutes` in reaction to avoid confusion?

Reply:
Yup, that and feature AKG -- I haven't had a chance to move the JSONPage stuff over, so till then would be best.

Reply:
One thing to note (and this relates to deprecation paths and their complexity) is that this sort of route sharing between old and new has led to a (lot) of complexity and some subtle bugs in latest batch of auctions code over the past year. This feature stuff is obviously way simpler, but yah, lets strive towards deprecation 😄